docs/website/docs/guides/deployment-configurations/gpu-rocm.md

```diff
@@ -87,12 +119,15 @@ on GPUs.
 
 --8<-- "docs/website/docs/guides/deployment-configurations/snippets/_iree-import-onnx-mobilenet.md"
 
-Then run the following command to compile with the `rocm` target:
+Then run the following command to compile with the `rocm` target backend:
+
+```shell hl_lines="5-6"
+# See the section below for information about HIP targets.
+IREE_HIP_TARGET=$(rocm_agent_enumerator | sed -n '2 p')  # e.g. gfx1100
```

Comment:
Instead of making this a variable, could we suggest to run the command and copy the output? This way the user has a chance of understanding the surrounding context IMO instead of blindly executing commands.

Reply:
Also, would it make sense to show the `--dump_devices` version here instead of using two different methods to find the target?

Reply:
Made a few adjustments. How's this look now? I did want to keep the example working with just copy/paste but I added another section showing `--dump_devices` output and elaborated on what `rocm_agent_enumerator` does.